Update README_de_DE.md

Alino001 opened the pull request:
Updated the German Translation to be more understandable (I know German, because i live in Germany).

PencilNavigator reviewed (changes requested):
Please make sure the translation is close to the original text, adding extra things is not needed.

Alino001 commented:
Ah ok, gonna change then

PencilNavigator commented on translations/README_de_DE.md:
```diff
@@ -17,9 +17,9 @@
 # Was ist Atlas?
 
-Atlas ist eine modifizierte Version von Windows, welche alle negativen Nachteile von Windows beseitigt, die sich negativ auf die Spieleleistung auswirken. Wir sind ein transparentes und quelloffenes Projekt, dass die Gleichberechtigung der Spieler anstrebt, unabhängig davon, ob Sie einen Kartoffel- oder einen Gaming-PC verwenden.
+Atlas ist eine modifizierte Version von Windows, welche alle negativen Nachteile von Windows beseitigt, die sich negativ auf die Spieleleistung auswirken. Wir sind ein transparentes und quelloffenes Projekt, dass die Gleichberechtigung der Spieler anstrebt, unabhängig davon, ob Sie einen Sehr alten- oder einen Gaming-PC/Laptop verwenden.
```
Delete "/Laptop"

PencilNavigator commented on translations/README_de_DE.md:
```diff
@@ -35,7 +35,7 @@ Obwohl unser Hauptaugenmerk auf der Leistung liegt, sind wir auch eine hervorrag
 ### **Privatsphäre**
 
-Atlas entfernt alle Arten von Tracking, die in Windows eingebettet sind, und implementiert zahlreiche Gruppenrichtlinien, um die Datenerfassung zu minimieren. Was außerhalb des Bereichs von Windows liegt, können wir jedoch nicht hinsichtlich des Datenschutzes verbessern, wie zum Beispiel Websites, die Sie besuchen.
+Atlas entfernt alle Arten von Tracking (Spyware), die in Windows eingebettet sind, und implementiert zahlreiche Gruppenrichtlinien, um die Datenerfassung zu minimieren. Was außerhalb des Bereichs von Windows liegt, können wir jedoch nicht hinsichtlich des Datenschutzes verbessern, wie zum Beispiel Websites, die Sie besuchen.
```
Delete "(Spyware)"

PencilNavigator commented on translations/README_de_DE.md:
```diff
@@ -51,7 +51,7 @@ Atlas strebt danach, so sicher wie möglich zu sein, ohne Leistungseinbußen zu
 ### **Debloated**
 
-Atlas ist stark entschlackt, vorinstallierte Anwendungen und andere Komponenten wurden entfernt. Trotz möglicher Kompatibilitätsprobleme wird dadurch die ISO- und Installationsgröße erheblich verringert. Funktionen wie Windows Defender usw. wurden vollständig entfernt. Diese Änderungen sind auf Gaming ausgerichtet, aber die meisten Arbeits- und Bildungsanwendungen funktionieren. [Was wir sonst no ...
+Atlas ist sehr stark modifiert. Vorinstallierte Anwendungen und andere Komponenten wurden entfernt. Trotz möglicher Kompatibilitätsprobleme wird dadurch die ISO- und Installationsgröße erheblich verringert. Funktionen wie Windows Defender usw. wurden vollständig entfernt. Diese Änderungen sind auf Gaming ausgerichtet, aber die meisten Arbeits- und Bildungsanwendungen funktionieren. [Was wir sonst ...
```
"heavily stripped down" not "modified"
(checked with DeepL Translator)

Alino001 commented:
Changed some stuff
Potato-PC = Kartoffel-PC 
The issue is in German you say Schlecht (bad) or Sehr alt (Very old)
Removed some added stuff
and Haupthema (main focus) is better than Hauptaugenmerk (that word is ancient)

PencilNavigator reviewed (approved):
Looks good.

Alino001 commented:
i fixed that “modified” issue

Xyueta reviewed (approved):
Approved.

Alino001 commented:
good
